pages/your_tasks/data_protection.md

```diff
@@ -23,6 +23,50 @@ faircookbook:
   url: https://w3id.org/faircookbook/FCB074
 ---
 
+## How do you ensure that your data is handled securely?
+
+### Description
+
+To protect your research data, code, and other information assets you should establish adequate Information security measures. Information security relies on combinations of technical and procedural measures that work together to ensure protection of the data at a suitable level. To note is that failure by humans to follow the procedural measures often poses a higher vulnerability to security risks than shortcomings of the technical measures. 
+
+### Considerations
+
+* Information security is usually described as containing three main aspects - **Confidentiality**, **Integrity**, and **Availability**.
+  * Confidentiality is about measures to ensure that data is kept confidential from those that do not have rights to access the data.
+  * Integrity is about measures to ensure that data is not corrupted or destroyed during storage or transmission.
+  * Availability is about measures that ensure access to the data is possible at the time of need or access does not require an inordinate amount of time.
+* What information security measures that need to be established should be defined at the planning stage (e.g. when doing a risk assessment for a GDPR Data Protection Impact Assessment). This should identify information security risks, and define measures to mitigate those risks. 
+* Balance the cost and effort for protecting your data at an appropriate level. For this, it is useful to classify the impact of failure for the three different CIA aspects to guide your choices. You should not spend more resources than necessary if the impact of failure is low enough that you can easily cope with it. Nor should you spend too few resources on protective measures for aspects where the impact is high. For example, the Availability aspect will probably differ a lot if your data is used in a clinical versus a research setting, where protecting against a system failure in the research setting can be a lot less resource demanding than in the clinical setting.
+* Ensure that everyone that works with the data knows what is expected of them, how the Information Security measures are to be implemented, and who is responsible for ensuring that they are followed and performed as agreed.
+* Be sure to contact the IT or Information security office at your institution to get guidance and support to address these issues. They should be able to provide you with suitable solutions for your needs.
+
+### Solution
+
+* Below is a far from exhaustive list of some suggested good practices that might be relevant depending on the level of protection required. Note that some of these might be quite costly. 
```

Comment:
Below is a non-exhaustive list of good practices that might...

Reply:
Done